.vscode/extensions.json

```diff
@@ -0,0 +1,25 @@
+{
+    "recommendations": [
+        "alefragnani.bookmarks",
+        "msjsdiag.debugger-for-chrome",
+        "kisstkondoros.vscode-codemetrics",
+        "yorkxin.coffeescript-support",
+        "drknoxy.eslint-disable-snippets",
+        "eamodio.gitlens",
+        "ms-vsliveshare.vsliveshare",
+        "karigari.chat",
+        "lostintangent.vsls-pomodoro",
+        "lostintangent.vsls-whiteboard",
+        "yzhang.markdown-all-in-one",
+        "bierner.markdown-emoji",
+        "eg2.vscode-npm-script",
+        "christian-kohler.npm-intellisense",
+        "buenon.scratchpads",
+        "jasonnutter.search-node-modules",
+        "asvetliakov.snapshot-tools",
+        "fabiospampinato.vscode-terminals",
+        "chrisbreiding.test-utils",
+        "britesnow.vscode-toggle-quotes",
+        "pflannery.vscode-versionlens"
+    ]
+}
```

Comment:
I feel like a lot of these are too opinionated. 

We should just recommend the ones that are essential for contributors to commit - like, they need eslint to work, coffescript support utils, test-utils, typescript utils, really any utilities that will catch things before the linter will in CI - cause a lot of them don't check CI later. 

I don't use any of these for example and would just be another file that may be confusing to contributors.

Reply:
yeah, i like the idea of recommending extensions, but maybe it should be done via documentation, so we can be like "here's what you NEED. here's what you might want."

Reply:
The intention was more to share recommended extensions with our team. It doesn't auto-install them or gets in the way (merge conflicts, etc.). But it does make it easier to discover and also install them. It is not much different than documentation, but this provides more value than documentation alone. With that said, I'm not opposed to adding it to the docs if it means it getting in or not getting in; I'm just not sure the difference between the two is worth the value trade-off.

I'm hardpressed to believe that contributors will find this confusing. I also doubt that the number of contributors to add to this would be overwhelming for us; issues/PRs opened, etc. Sure, I could be wrong about this, but I'm not convinced as of now this is the case and I'm unsure if anyone would be able to definitively say it is one way or another. Due to us simply not knowing, I'd rather add value and iterate on it later (remove it, etc.) than simply never add it because we think it could be a problem in a nebulous way.

I suppose if the list were to grow to an enormous number then that could be problematic, but that is a future problem that we may never even encounter. I doubt the list will continue to grow at a steady rate over time. I imagine there is a rough limit that we will naturally arrive at that is not unmanageable.

Reply:
If this is for our team only, then there should be some discussion on the extensions that are essential for our team, because this list seems very arbitrary to me. I only use 2 of these extensions for example, so this just seems like your personal list of extensions which is just noise in our repo if no one ever uses.

Reply:
I can agree with that for sure. These are some that I have found helpful and/or learned about using with the Cypress codebase. What are the two you use that are also on the list? We can add/remove as necessary. Also, are there any that you find helpful that are not on the list?

Reply:
I can agree with that for sure. These are some that I have found helpful and/or learned about using with the Cypress codebase. What are the two you use that are also on the list? We can add/remove as necessary. Also, are there any that you find helpful that are not on the list?